config/300-configuration.yaml

```diff
@@ -21,7 +21,13 @@ metadata:
     knative.dev/crd-install: "true"
 spec:
   group: serving.knative.dev
-  version: v1alpha1
+  versions:
+  - name: v1alpha1
+    served: true
+    storage: true
+  - name: v1beta1
+    served: true
```

Comment:
Some tests would be nice to know that we are actually serving v1beta1.

Reply:
We discussed this a bit.  My bias is to try to avoid generating clients for now.

To that end I am going to add the following tests for now:
1. Use the dynamic client to send a `runLatest` resource to v1beta1 endpoint and verify that it is rejected.
2. Use the dynamic client to fetch (via v1beta1) a freshly submitting (over v1alpha1, `runLatest`) resource and verify that it looks right.

Reply:
Ok, we should have both tests.